swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -37,6 +37,9 @@
  * ****************************************************************************/
 package com.nqadmin.swingset;
 
+import static com.nqadmin.swingset.models.AbstractComboBoxListSwingModel.addEventLogging;
```

Comment:
I used the "organize imports" command in Eclipse if I was left with unnecessary imports following coding changes. It changes the sorting.